README.md

```diff
@@ -127,6 +127,43 @@ Staged changes can be reviewed like this:
 $ nixpkgs-review wip --staged
 ```
 
+## Using nix-review in scripts
+
+After building, `nixpkgs-review` will normally start a `nix-shell` with the
+packages built, to allow for interactive testing. To use `nixpkgs-review`
+non-interactively in scripts, use the `--no-shell` command, which can allow for
+batch processing of multiple reviews or use in scripts/bots.
+
+Example testing multiple unrelated PRs:
+
+```bash
+for pr in 807{60..70}; do
+    nixpkgs-review pr --no-shell $pr && echo "PR $pr succeeded" || echo "PR $pr failed"
+done
+```
+
+Example composing `nixpkgs-review` inside a script and using a token from
+[hub](https://github.com/github/hub) to post a comment on the PR with results:
+
+```
+review_pr_with_comment() {
+    if ~/src/nixpkgs-review/result/bin/nixpkgs-review pr --no-shell $1; then
+        msg="nixpkgs-review succeeded"
+    else
+        msg="nixpkgs-review failed"
```

Comment:
What do you think about using?

msg=$(cat ${XDG_CACHE_HOME:-HOME}/.cache/nix-review/pr-$1/report.md)

Reply:
I think that's a great idea; that looks a lot more full featured than the bare placeholder example comment I was putting here.

That said, I can't quite get the quoting in the `$msg` to work ... do you know how to sanitize it before giving to curl?